Makefile

```diff
@@ -1,14 +1,9 @@
 REPORTER = $(if $(CI),spec,dot)
 
-ifeq ($(TRAVIS),)
 build: node_modules/
-else
-PATH := $(PWD)/phantomjs/bin:$(PATH)
-build: node_modules/ phantomjs/bin/phantomjs
-endif
 
 test: node_modules/ build lint
-	phantomjs ./node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js \
+	./node_modules/phantomjs-prebuilt/bin/phantomjs ./node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js \
```

Comment:
Yeah, I actually hit that bug. fcd056b5b9a5befb7483e16bdd1ff2f997ec93cd made the change. Appears that `npm install` only rebuilds binstubs on packages it installs and does not rebuild them all. Though, I'm sure we could maybe pass some extra flag to it, but this seems fine.